### PR TITLE
EN-1933 fix resource parameter handling (4.3.5)

### PR DESCRIPTION
**Pull request by eugeniosant**

### CODE REVIEW

**Comment by eugeniosant** [comment]
retest this please

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
Something went wrong. Investigate!

File: admin-console/src/test/java/com/agiletec/plugins/jacms/apsadmin/content/TestContentAdminAction.java [hunk 1]
```diff
@@ -39,6 +39,9 @@ protected void setUp() throws Exception {
 
     public void testOpenIndexProspect() throws Throwable {
         String result = this.executeOpenIndexProspect("admin");
+        synchronized (this) {
```
Comment (joewhite101) [inline admin-console/src/test/java/com/agiletec/plugins/jacms/apsadmin/content/TestContentAdminAction.java]: Don't like this ... What is going on that this is required?

**Comment by entando-service** [comment]
Something went wrong. Investigate!

File: admin-console/src/test/java/com/agiletec/plugins/jacms/apsadmin/content/TestContentAdminAction.java (hunk 1 above)
Comment (eugeniosant) [reply admin-console/src/test/java/com/agiletec/plugins/jacms/apsadmin/content/TestContentAdminAction.java]: reverted wrong commit

**Comment by entando-service** [comment]
All is well

**Pull request merged**